src/content/docs/errors-inbox/version-tracking.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Error tracking with errors inbox'
+metaDescription: "Errors inbox is an error tracking solution designed to give you the tools to proactively detect, triage, and resolve errors across your full application stack."
+---
+
+import errorsinboxFilterErrorsbyVersions from 'images/errors-inbox_screenshot-crop_filter-errors-by-versions.webp'
+
+Errors inbox version tracking allows developers to monitor each deployment to check for accuracy and success. Lets say you and your team an alert that says one of your entities has a spike in error rates. You identify the [source of the error](docs/tutorial-error-tracking/respond-outages/) and decide to release a canary deployment. You can use version tracking to monitor your fix as well as the previous deployments to determine if you solved the problem or if you need to go back to the drawing board. 
+
+<Steps>
+<Step>
+## Instrument version tracking on your entities. 
+
+To track errors for each deployment in APM and OpenTelemetry applications, make sure sure  the following event fields are checked:
+
+* `service.version` 
+* `tags.service.version`
+* `tags.releaseTag`
+* `tags.commit`
+
+For mobile applications, the event field is `appVersion`.
+
+For browser applications, the event field is `application.version`.
+</Step>
+
+<Step>
+## Configure fields 
+
+To capture data from our version tracking, you must configure fields for relevant entities:
+
+* For OpenTelemetry entities, configure `service.version`.
```

Comment:
will each of these bullets have links to the relevant section in the docs?

Reply:
Yes!